python/desc/skycatalogs/objects/base_object.py

```diff
@@ -305,7 +309,9 @@ def get_gsobject_components(self, gsparams=None, rng=None):
                 n = round(n*20.)/20.
                 obj = galsim.Sersic(n=n, half_light_radius=hlr,
                                     gsparams=gsparams)
-            shear = galsim.Shear(q=b/a, beta=beta)
+
+
```

Comment:
Maybe put a comment here that we still need to understand whether the gcr-catalogs position angle needs that extra +90 degrees, amounting to those minus signs for `e1` and `e2`?   Those minus signs may end up as yet another special case to reproduce DC2 galaxy shapes and orientations.

Reply:
I've added something.   Please let me know if it's not what you had in mind.

Reply:
Yes, looks good.